pimsviewer/viewer.py

```diff
@@ -656,6 +704,11 @@ def to_pixmap(self):
     def to_clipboard(self):
         clipboard = QtWidgets.QApplication.clipboard()
         clipboard.setPixmap(self.to_pixmap())
+
+    @staticmethod
+    def _get_all_files_in_dir(directory):
+        return [f for f in listdir(directory) if isfile(join(directory, f))]
```

Comment:
Maybe good idea to cache this? An easy way would be a memoizer decorator.
And sort it, using

```
from pims.utils.sort import natural_keys
sorted(filepaths, key=natural_keys)
```